cleanlab_studio/internal/util.py

```diff
@@ -63,3 +64,131 @@ def check_none(x: Any) -> bool:
 
 def check_not_none(x: Any) -> bool:
     return not check_none(x)
+
+
+def _get_autofix_default_params() -> dict:
+    """returns default percentage-wise params of autofix"""
+    return {
+        "ambiguous": 0.0,
+        "label_issue": 0.5,
+        "near_duplicate": 0.2,
+        "outlier": 0.5,
+        "confidence_threshold": 0.95,
+    }
+
+
+def _get_autofix_defaults(cleanset_df: pd.DataFrame) -> dict:
```

Comment:
TODO: Studio team should move this function to backend of the app so it happens on server (eventually should be used in web app too)